.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.4
+current_version = 1.0.5
```

Comment:
I think bumping the version should be done out of this PR. 
For example, we shouldn't be releasing from a tag in a brahch; also, CHNAGELOG doesn't have a date yet.